docs/config.rst

```diff
@@ -1,11 +1,606 @@
 Configuring CumulusCI
 =====================
 
-cumulusci.yml
--------------
+CumulusCI merges multiple `YAML <https://yaml.org/>`_ files that allow for configuration to occur at several distinct scopes. All of these files have the same name, ``cumulusci.yml``, but live in different locations on the file system. (See `configuration scopes`_ for further detail.)
 
-Global overrides
-----------------
+The `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file contains the default configuration stored within CumulusCI itself. This is the location of the standard library of tasks and flows that come with CumulusCI.
 
-Local overrides
----------------
+The project ``cumulusci.yml`` file is located in the ``project`` diretory. You can override aspects of the universal ``cumulusci.yml`` file here by defining project dependencies, configuring new tasks and flows, customizing standard tasks and flows to meet your project's needs, and so much more! 
+
+    .. Important:: When ``cumulusci.yml`` is referenced in this documentation, it refers to the project ``cumulusci.yml`` file unless otherwise specified.
+
+
+
+``cumulusci.yml`` Structure
+---------------------------
+
+There are six top-level sections of a ``cumulusci.yml`` file.
```

Comment:
```suggestion
These top-level sections make up a ``cumulusci.yml`` file.
```
When introducing a list, avoid stating the number of items in the list because it might change. I've called out this style issue in another PR. Please be on the lookout for it. https://confluence.internal.salesforce.com/display/DOCTEAM/lists

Incidentally, also be on the lookout for "There are." It's a weak way to start off a sentence and usually a tip-off that stronger, less wordy writing is possible.